api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/PractitionerReferenceTranslatorUserImpl.java

```diff
@@ -42,7 +42,9 @@ public User toOpenmrsType(Reference reference) {
 		if (reference == null) {
 			return null;
 		}
-		if (!getReferenceType(reference).equals("Practitioner")) {
+		
+		String type = reference.getType();
+		if (type != null && !type.equals("Practitioner")) {
 			throw new IllegalArgumentException("Reference must be to an User not a " + reference.getType());
```

Comment:
@varung-31Thanks for working on this,what if we also used type instead of reference.getType() around here? cc   @ibacher